sdk/python/feast/type_map.py

```diff
@@ -38,6 +39,44 @@
 from feast.value_type import ValueType
 
 
+def feast_value_type_to_python_type(field_value_proto: ProtoValue) -> Any:
+    """
+    Converts field value Proto to Dict and returns each field's Feast Value Type value
+    in their respective Python value.
+
+    Args:
+        field_value_proto: Field value Proto
+
+    Returns:
+        Python native type based on Feast Value Type
+    """
+    field_value_dict = MessageToDict(field_value_proto)
+
+    for k, v in field_value_dict.items():
+        if k == "int64Val":
+            return int(v)
+        if k == "bytesVal":
+            return bytes(v)
+        if (k == "int64ListVal") or (k == "int32ListVal"):
+            return [int(item) for item in v["val"]]
+        if (k == "floatListVal") or (k == "doubleListVal"):
+            return [float(item) for item in v["val"]]
+        if k == "stringListVal":
+            return [str(item) for item in v["val"]]
+        if k == "bytesListVal":
+            return [bytes(item) for item in v["val"]]
+        if k == "boolListVal":
+            return [bool(item) for item in v["val"]]
+
+        if k in ["int32Val", "floatVal", "doubleVal", "stringVal", "boolVal"]:
```

Comment:
Is there a reason why these types can't be converted into python native types?

Reply:
They can and are automatically converted as shown [here](https://developers.google.com/protocol-buffers/docs/proto3#json). So don't need to explicitly do so.